start-from-zero/index.css

```diff
@@ -246,21 +350,208 @@ textarea {
 
 #select {
     float: right;
+    -webkit-appearance: none; 
+    -moz-appearance: none;
+    appearance: none;   
+    background-image: url(assets/down-arrow.png);
+    background-repeat: no-repeat;
+    background-position: 90% 50%;
+    background-size: 13px;
+    position: relative;
+    height: 20px;
+    
+}
+
+
+
+textarea {
+    width: 99.7%;
+    margin: 0 auto;
+    padding: 0;
+}
+
+
+#footer-new {
+    position: relative;
 }
 
 fieldset {
+    margin: 30px 0;
+}
+
+.fieldset-wrap {
+    padding: 10px 10px;
+}
+
+fieldset legend {
+    padding: 0 20px;
+}
+
+#inner-li {
+    height: 100%;
+}
+
+.ultimate-wrap {
     width: 100%;
+    margin: 0 auto;
 }
 
-#footer-new {
+.outer-ul, .down-ul {
+    height: 100%;
+    display: inline-flex;
+    flex-direction: row;
+    justify-content: space-between;
+}
+
+.down-ul {
     position: relative;
-    
+    bottom: 80px;
+}
+
+#height-100 {
+    height: 100%;
+}
+
+.inner-ul li {
+    height: 100%;
+    line-height: 80px;
+    position: relative;
+    z-index: 1000;
+}
+
+
+
+.position-div {
```

Comment:
Same comment about naming. Can we come up with something more meaningful? This doesn't tell us much about the purpose / usage.